index.html

```diff
@@ -210,8 +273,8 @@ <h2 id="2NormRefs">Normative references</h2>
 information exchange between systems &mdash; High-level data link
 control (HDLC) procedures &mdash; Frame structure</i>.</p>
 
-<!-- Maintain a fragment named "2-ISO-8859-1" to preserve incoming links to it -->
-<p class="NormRefDef" id="2-ISO-8859-1">ISO/IEC
+<!-- Maintain a fragment named "2-ISO 8859-1" to preserve incoming links to it -->
+<p class="NormRefDef" id="2-ISO 8859-1">ISO/IEC
```

Comment:
The old fragment had the dash in it. We need to preserve it.

Reply:
Oops my bad, a search-and-replace that clobbered something it shouldn't

Reply:
I think we still want this id to have a dash, right?

Reply:
It is in the section that will be deleted - the old, manual, normative references section.

I can see someone linking to a particular spec section, (like a chunk definition) or to a defined term, but doubt there are many/any links to an entry in the references. They would just link directly to that reference, surely.